doc/howto_build.md

```diff
@@ -92,7 +92,7 @@ available on all supported platforms.
 
     GCOV                Support for code coverage using libgcov.
 
-                        This build variant requires libgcov and is not availble
+                        This build variant requires libgcov and is not available
```

Comment:
This moves the line beyond the 80 columns limit. Please change the formatting to stay within 80 columns.

Reply:
it's exactly 80. should it be 79? in this case there are are lines which are 80, even in this file..

Reply:
It should be 79 since we start counting at zero.  There are other instances where we go to 80 instead of 79.  You may leave this one for me to fix if you want.

Reply:
updated